### ISSUE TITLE
Full width sections in container

### ISSUE THREAD

**Issue by etwordpress01** [ThemeFuse/Brizy#65]
Hi, everyone, I have a quick question. We are using Brizy page builder in our theme and we don't want to use Brizy template. We used WordPress default template. My Question is how we can make section full width while themes used containers? I think there should be a solution same like Visual Composer. Sections should get full width in container as well. 

Please check both Visual Composer and Brizy sections

Any solution for it? help me please

![1](https://user-images.githubusercontent.com/31212934/40607148-1da86930-6281-11e8-84ed-0e8bfcb5efe9.png)
![2](https://user-images.githubusercontent.com/31212934/40607149-1dde76a6-6281-11e8-98ae-09cf1228322f.png)

Section options should something like below

![3](https://user-images.githubusercontent.com/31212934/40607317-8ef7eef8-6281-11e8-9f65-30ce8df7160b.png)

**Comment by bagrinsergiu** [comment]
Hi @etwordpress01,

Thanks for letting us know, we are looking into it.

Sergiu

**Comment by bagrinsergiu** [comment]
Thanks for your efforts! Much appreciated.

Can you give me WP credentials via Facebook Chat: http://dl4.joxi.net/drive/2018/05/10/0028/0732/1860316/16/cb53437dfd.jpg

Thank you.

**Comment by etwordpress01** [comment]
Hi, Let me share credentials via Facebook chat. This feature is required for everyone. Here is another ticket in which someone ask same question : https://github.com/ThemeFuse/Brizy/issues/51

Thanks

**Comment by bagrinsergiu** [comment]
@etwordpress01 - Issue #51 is in our to do list for this week. We plan to solve this with next update.  Can you give me WP details via FB.

Thanks

**Comment by yakiaz** [comment]
Is there any update around this issue ? we are facing the same requirement and issue. THX a MIL @bagrinsergiu

**Comment by allerup** [comment]
I am also looking forward to a solution to this issue :)

**Comment by bagrinsergiu** [comment]
Nope. I made a mistake to promise in issue #51. I don't know how to implement this feature in editor and i don't understand why this is necessary. 

I see a better solution just use a full-width template.

**Comment by etwordpress01** [comment]
Hello @bagrinsergiu
This is most important fix in your page builder. Unyson also doesn't give solution for it. But i added this fix in Unyson for my own use. Visual Composer give options to stretch section to make it full width, no matter themes uses containers( boxed/specific width site ). Section should have option to make it full width instead of making complete template full width. End users don't know that they have to select full width section while creating a page.

Let me give you an example again. please view screenshots

![a](https://user-images.githubusercontent.com/31212934/41716587-5cf164fa-7570-11e8-9b61-4a1b12b3b518.png)
![b](https://user-images.githubusercontent.com/31212934/41716595-643c5cc4-7570-11e8-99fa-426a55f0776a.png)

![c](https://user-images.githubusercontent.com/31212934/41716605-6d6b4f08-7570-11e8-8bb5-cffb7b0e3788.png)

Hopefully this will clear that why every your customers need this option in Brizy page builder. This is just a js trick.  Let me know if i can help?

Thanks

**Comment by bagrinsergiu** [comment]
I’m closing this issue to keep the issue database slim. Feel free to re-open if I’ve misunderstood you.

Thank you report.
Sergiu

**Comment by etwordpress01** [comment]
Hi, This issue has not resolved yet. I can explain it in detail if you want. But this is most useful feature in Brizy plugin. Hopefully in future other users will raise this issue.

Template hack is something else, If Visual Composer author can add full width section options within container then why you don't? 

Thanks

**Comment by heenaab** [comment]
Hi,
I'm having the same issue, a cannot seem to figure out a way to make sections full width when editing with brizy. Is there any solution to this yet ?
Thanks

**Comment by newdevgirl** [comment]
My website auto-updated to Wordpress 5.0.2 and when I opened my non-Brizy Home Page, it defaulted to a Brizy Template and took away my full width site design on that page. It looks like stacked content now.
<img width="1336" alt="screenshot 2018-12-30 13 59 26" src="https://user-images.githubusercontent.com/42420306/50551770-3bf49800-0c3b-11e9-868d-434f36b761b7.png"> Now I can't figure out how to get back my customized Twenty Seventeen theme with full-width autoplaying video. Please advise.

**Comment by robbieroche** [comment]
I have the same issue. Has there been any updates?
